packages/react-kitchen-sink/src/react-query/observers/__tests__/querySnapshotQueryClientObserver.test.ts

```
@@ -0,0 +1,46 @@
+import { type QueryClient } from '@tanstack/react-query'
```

ESLint

[error] Filename is not in kebab case. Rename it to `query-snapshot-query-client-observer.test.ts`.

(unicorn/filename-case)